### DISCUSSION TITLE
GPU only makes it faster, not better quality, right?

### DISCUSSION THREAD

**Discussion by HubKing** [openai/whisper#2349] [Q&A] [answered]
My PC has an Intel Arc, which currently doesn't seem to be supported. I used the "Large" model to transcribe some movie.  I compared the result of Whisper and of Riverside. The result was mostly the same, but "Large" seemed worse than Riverside's. I only did 1:30 transcription, because running it on CPU was slow, but there were two errors in Whisper.

Riverside: Désolé pour ce long silence.
Whisper:  désolé pour ce long soir.  Je suis en silence.

Riverside: les grattes-ciels qui brillent au loin
Whisper: les grades ciels qui brillent au loin

"Large" is the best that OpenAI can do and even if I use a good GPU, the quality doesn't get better, does it?

**Comment by ryanheise** [comment, marked as answer]
That is correct, GPU only makes it faster, not better quality.

There's certainly excellent competition out there from commercial products, and Deepgram (which Riverside uses under the hood) is one of those. As you can see, Deepgram sometimes makes similar (and some different) errors to Whisper, and no transcription model is currently at a 0% error rate. One factor that matters to some people is that Whisper is open source which gives you a bit more control over things (such as fine tuning). But you can always compare the alternatives and pick what suits your needs.

**Comment by ryanheise** [comment]
> My PC has an Intel Arc, which currently doesn't seem to be supported.

If you want to experiment, there is also a PR to support this here: #1362 . Although it has some unresolved conflicts with the latest Whisper, you can see the essential changes below if you want to experiment with it:

https://github.com/openai/whisper/pull/1362/files
